docs/dataloader_options.md

```diff
@@ -0,0 +1,12 @@
+# Dataloader implementation options
+This doc contains our options for implementing a [dataloader](https://github.com/graph-gophers/dataloader). It has been created as a result of [#480](https://github.com/creativesoftwarefdn/weaviate/issues/480).
+
+## Background
+Our dataloader implementation exists as middleware between the GraphQL endpoint's resolvers and the database connector. It is written as a connector, implementing the [BaseConnector](https://github.com/creativesoftwarefdn/weaviate/blob/d70782a6619315b85518f7e79791fe70ed4698dd/connectors/database_connector.go#L30) interface.
+
+## Implementation options
+
+![getgraph options dataloader](https://user-images.githubusercontent.com/9214481/45943227-d19ddd00-bfe5-11e8-87a1-bfbeaa6b7262.png)
```

Comment:
I would not show all three options. But only the one that you've chosen. Also, option C does not reflect what is in the issue: https://github.com/creativesoftwarefdn/weaviate/issues/494#issuecomment-423966549